storybook/src/brand/design-tokens/links.docs.mdx

```diff
@@ -16,11 +16,11 @@ Our default links have the following styles:
 | CSS variable                                       | Value                                | Description                                                                       |
 | :------------------------------------------------- | :----------------------------------- | :-------------------------------------------------------------------------------- |
 | `var(--ams-links-color)`                           | `var(--ams-color-interactive)`       | The default colour for links is our interactive colour.                           |
-| `var(--ams-links-text-decoration-thickness)`       | `var(--ams-border-width-m)`          | Link underlines are our medium border width.                                      |
-| `var(--ams-links-text-underline-offset)`           | `0.3333em`                           | The underline is positioned further below the text.                               |
+| `var(--ams-links-text-decoration-thickness)`       | `0.125rem`                           | Link underlines are 2 pixels thick (respecting users’ custom font size settings). |
```

Comment:
I don't think we need the part between parenthesis, otherwise we could / should add it to all our `rem` value docs.